src/main/java/batman/tasks/Event.java

```diff
@@ -0,0 +1,45 @@
+package batman.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+import batman.parser.DateUtil;
+
+/**
+ * A Task encapsulating an event with a date and time.
+ */
+public class Event extends Task {
+
+    protected LocalDateTime at;
+
+    /**
+     * @param isDone Whether the task is done.
```

Comment:
Shouldn't there be a method description in the JavaDoc comments for all your methods? Good job for your params, throws and return sections of the JavaDoc.